src/util/entities/User.ts

```diff
@@ -333,6 +337,15 @@ export class User extends BaseClass {
 		}
 	}
 
+	public get tag(): string {
+		const { pomeloEnabled } = Config.get().general;
+
+		// if pomelo is enabled, global_name should be set
+		return pomeloEnabled
```

Comment:
Pomelo is Discord inc's codename, should be called `uniqueUsernames` instead

Reply:
good point